doc/sphinx/proofs/writing-proofs/equality.rst

```diff
@@ -628,16 +628,34 @@ which reduction engine to use.  See :ref:`type-cast`.)  For example:
 
 .. tacn:: cbn {? @reductions } @simple_occurrences
 
-   :tacn:`cbn` was intended to be a more principled, faster and more
-   predictable replacement for :tacn:`simpl`.
-
-   The main difference between :tacn:`cbn` and :tacn:`simpl` is that
-   :tacn:`cbn` may unfold constants even when they cannot be reused in recursive calls:
-   in the previous example, :g:`succ t` is reduced to :g:`S t`.
+   :tacn:`cbn` is a variant of :tacn:`cbn` implemented as an abstract machine.
```

Comment:
Not sure this is much more useful to the manual reader than the previous comment.
Typically, the user still don't know when and why she should prefer `cbn` over `simpl` (or vice versa).

Reply:
Yes! And to be honest, I still don't know myself. Hopefully, the ongoing discussions will help.

Reply:
Shouldn't you be comparing simpl and cbn here?  I don't see how "cbn is a variant of cbn" makes any sense.

Reply:
@herbelin apparently, the discussion is not happening / leading to any conclusion, why not reverting that sentence and merge the remaining of the PR?

Reply:
Let's merge as it is without this sentence since the discussion is not progressing on it.